.gitignore

```diff
@@ -1,2 +1,4 @@
 bower_components*
 bower-1.x.json
+build/
+.idea
```

Comment:
This should probably not be in here - it belongs in a global `.gitignore` config on a dev's local machine.